dfastmi/batch/AnalyserDflowfm.py

```diff
@@ -0,0 +1,376 @@
+from pathlib import Path
+from dfastmi.batch.SedimentationVolume import comp_sedimentation_volume
+from dfastmi.kernel.core import main_computation, dzq_from_du_and_h
+from dfastmi.batch.DflowfmLoggers import AnalyserDflowfmLogger
+from dfastmi.batch.OutputDataDflowfm import OutputDataDflowfm
+from dfastmi.batch.XykmData import XykmData
+from dfastmi.io.GridOperations import GridOperations
+from dfastmi.kernel.typehints import Vector, BoolVector
+from shapely.geometry.linestring import LineString
+
+import numpy
+import os
+from typing import Any, Dict, Optional, TextIO, Tuple, Union
+
+class AnalyserDflowfm():
+    """
+    Class that analyses the Dflowfm data.
+    """
+    
+    _logger : AnalyserDflowfmLogger
```

Comment:
is logger the right name here? What about reporter.. Logger confuses me by remarks like : "We should use the default pylogger package"... etc. It is good to have this class but also seems sort of confusing

Reply:
it started with moving the ApplicationSettingsHelper.log_text to these classes, after that the print statements, aggreed that logger might not be the right naming, lets discuss on monday.